### DISCUSSION TITLE
Do you have any approximate plans for releasing next versions?

### DISCUSSION THREAD

**Discussion by ascensus-mdb** [twbs/bootstrap#31621] [General] [answered]
It seems the topic of the V5 is completely dead. I understand times are tough and everyone has more important things to do, but can you please tell if you have any plans to release next v5 versions in the near future?

Best regards and thanks for all your hard work

**Comment by Deckluhm** [comment]
Not an answer to your question (since there is no date for v5) but you can watch this repo for updates about v5 release: https://github.com/twbs/release

**Comment by mdo** [comment, marked as answer]
Hoping to push out alpha 2 next week :).
